roxie/ccd/ccdmain.cpp

```diff
@@ -1526,6 +1529,18 @@ int CCD_API roxie_main(int argc, const char *argv[], const char * defaultYaml)
                 writeSentinelFile(sentinelFile);
 #endif
                 DBGLOG("Startup completed - LPT=%u APT=%u", queryNumLocalTrees(), queryNumAtomTrees());
+                if (topology->getPropBool("expert/@profileStartup", false))
+                {
+                    startupTracer.stop();
+                    Owned<IFile> iFile = createIFile("startuptrace.svg");
```

Comment:
Should the filename be configurable - so you could write the file to a persistent mount point that you can access from a cloud system?  expert/@profileStartupFilename?